packages/babel-preset-env/test/fixtures/debug/browserslists-last-2-versions-not-ie/stdout.txt

```diff
@@ -0,0 +1,31 @@
+@babel/preset-env: `DEBUG` option
+
+Using targets:
+{
+  "android": "77",
+  "chrome": "77",
```

Comment:
Recently the chrome 49 usage just drops to 0.5% and therefore excluded from browserslists defaults `> 0.5%` now.